soliket/xcorr/limber.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pdb
+
+oneover_chmpc = 1. / 2997.92458
```

Comment:
This is ok for the moment. However, if this constant is needed by other theory calculators in soliket, it would be a good idea to create a constant.py module that can be shared by other components.